src/components/Blogposts.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import posts from '../data/posts.json';
+import {
+    Switch,
+    Route,
+    Link
+} from 'react-router-dom';
+import Blog
+    from "./Blog";
+
+
+export default function Blogposts({isAuthenticated}) {
+    const numberOfPosts = posts.length;
+
+    let listBlogposts = []
+    for (let i = 0; i < numberOfPosts; i++) {
+        let blogLink = "/blog/" + posts[i].id;
+        listBlogposts.push(<li><Link to={blogLink}>{posts[i].title}</Link></li>);
+    }
+
+    // console.log(posts);
+    return (
+        <>
+            {isAuthenticated &&
+            <>
+                <h1>Blog
+                    overzichtspagina</h1>
+                <h2>Aantal
+                    blogposts: {numberOfPosts}</h2>
+                <ul>{listBlogposts}</ul>
+                <h1>{posts[4].title}</h1>
+            </>
+            }
+            <Switch>
```

Comment:
Deze zou ik dus verplaatsen naar de hoofdpagina waar al je routing staat. Ik zag ook in je comment staat dat het niet was gelukt met de blog weergeven. Wat ik zo vlug kan zien hoef je alleen nog maar je component toe te voegen aan je route en deze te verplaatsen naar je mainpage. (eigenlijk zoals je het ook hebt gedaan met `Blogpost`